manifest.yml

```diff
@@ -5,7 +5,7 @@ description: |-
 
 dependencies:
   - name: "FreeRTOS-Kernel"
-    version: "V10.4.3"
+    version: "V10.4.4"
     repository:
       type: "git"
       url: "https://github.com/FreeRTOS/FreeRTOS-Kernel"
```

Comment:
Could you add the "path" attribute here to specify the relative path of the submodule?

Reply:
Done.